### PR TITLE
newer-tanooki: Add version 1.0

### PR DESCRIPTION
**Pull request by UnicorNora**

### CODE REVIEW

File: bucket/newer-tanooki.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0",
+    "description": "A NewerSMBW Save Editor.",
+    "homepage": "https://github.com/RedStoneMatt/NewerTanooki",
+    "url": "https://github.com/RedStoneMatt/NewerTanooki/releases/download/v1.0/NewerTanooki.rar",
+    "hash": "cbab72a7644bae3bf9f5ef901a85709f0f2e91b58fc6dea6f0a010073164877b",
+    "license": "Visable Source",
```
Comment (BlackPowerade) [inline bucket/newer-tanooki.json:7]:
```suggestion
    "license": "GPL-3.0-only",
```
NewerTanooki's license has been set

**Review by BlackPowerade — CHANGES_REQUESTED** [review]

File: bucket/newer-tanooki.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0",
+    "description": "A NewerSMBW Save Editor.",
```
Comment (Calinou) [inline bucket/newer-tanooki.json:3]:
```suggestion
    "description": "NewerSMBW save editor",
```

File: bucket/newer-tanooki.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            "NewerTanooki"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/RedStoneMatt/NewerTanooki/"
+    },
+    "autoupdate": {
+        "url": "https://github.com/RedStoneMatt/NewerTanooki/releases/download/v$version/NewerTanooki.rar"
+    }
+}
```
Comment (Calinou) [inline bucket/newer-tanooki.json:21]:
```suggestion
}

```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**